src/lazy/binary/encoded_value.rs

```diff
@@ -89,9 +84,9 @@ pub(crate) struct EncodedValue<HeaderType: EncodedHeader> {
     pub length_length: u8,
     // The number of bytes used to encode the value itself, not including the header byte
     // or length fields.
-    pub value_length: usize,
+    pub value_body_length: usize,
```

Comment:
🗺️ Several fields/methods with `value` in the name do not include information about the value's opcode or its trailing length. I've renamed them to make this more explicit.